sources/src/parser/nodetypes.cpp

```diff
@@ -241,6 +241,7 @@ namespace citygml {
                 INITIALIZE_NODE( BLDG, ConsistsOfBuildingPart )
 
                 INITIALIZE_NODE( BLDG, Lod0RoofEdge )
+                INITIALIZE_NODE(BLDG, Lod0FootPrint)
```

Comment:
細かいですが統一させるためにスペース追加お願いします。
```suggestion
                INITIALIZE_NODE( BLDG, Lod0FootPrint )
```